### DISCUSSION TITLE
Revenge squad still only lasts 60 seconds

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#13822] [Bug reports]
### What happened?

https://github.com/FakeFishGames/Barotrauma/issues/13466

![image](https://github.com/FakeFishGames/Barotrauma/assets/37850646/bbca694d-7773-4313-834e-86ca959dab53)

Max strength of the affliction hasn't been increased.
Duration only works if the affliction isn't reduced by `strengthchange`

### Reproduction steps

1. give revenge squad talent to a crew member
2. kill
3. observe that affliction still only lasts 60 seconds

### Version

v1.4.4.1 (Blood in the Water Update)

**Comment by Regalis11** [comment]
Whoops, fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/2fb30b1e27c56c321fa6c1558c5bd8ef91311e5f
